### DISCUSSION TITLE
deck.gl showcase suggestion: Where I've Flown

### DISCUSSION THREAD

**Discussion by sebastiaanh** [visgl/deck.gl#5088] [Show and tell]
Thanks for this great library! I'm wondering if you'd be interested to add my site to the showcase: https://whereiveflown.com

Where I've Flown allows you to easily create your own 3D map of all flights you’ve ever taken, and share the resulting URL with anyone you want. It is similar to "where I've been" maps, and there is no registration needed.

The app uses the ArcLayer via the MapboxLayer for very smooth results.

Previews:
![Preview 1](https://user-images.githubusercontent.com/73762410/97791953-4e91dc80-1bd8-11eb-96bc-c585dc461295.jpg)
![Preview 2](https://user-images.githubusercontent.com/73762410/97791957-55b8ea80-1bd8-11eb-8c58-7f8844efb421.jpg)

**Comment by Pessimistress** [comment]
Thank you for the suggestion! How is this different from the "FlightMapper" that's already in https://deck.gl/showcase?

**Comment by sebastiaanh** [comment]
It seems that FlightMapper is more targeted for pilots, plus it requires an account. It's true that both sites use the ArcLayer and thus have similar visualizations. A difference for my site could be that it also uses flat arcs [(example)](https://whereiveflown.com/#aesecv3ecvbhh~2mhi~ynearf3cot3vxe1vxetdj~bcl~all2~tkd~bggutp2~mle~gue1trf1bwh1+1), and that it demonstrates the MapboxLayer (e.g. fluid zooming).
